packages/react/src/primitives/Icon/icons/IconBoxArrowUpRight.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { classNames } from '@aws-amplify/ui';
+import { ComponentClassName } from '@aws-amplify/ui';
```

Comment:
This should only be a single import line

Reply:
Unresolved, there are two import lines for the same package still. In the future, please refrain from resolving a conversation until the feedback has been addressed

Reply:
Looks like there are two import lines for the same package across multiple primitives, I'll make sure to make it one line when done with other updates on this PR.